### PR TITLE
added loglines for testing

### PR DESCRIPTION
**Pull request by ajayOO8**
### Description
Added loglines for #281 

### Tests
`make test`

### CODE REVIEW

File: clusterman/aws/markets.py [hunk 1]
```diff
@@ -394,10 +398,12 @@ def fetch_instance_type_from_aws(instance_type: str) -> InstanceResources:
     disk_size = res.get("InstanceStorageInfo", {}).get("TotalSizeInGB", None)
     gpu_size = res.get("GpuInfo", {}).get("Gpus", [{}])[0].get("Count", 0)
 
+    logger.info(f"located instance-type details: {vcpu_count}, {mem_size}, {disk_size}, {gpu_size}")
     return InstanceResources(vcpu_count, mem_size, disk_size, gpu_size)
 
 
 def get_instance_type(instance_type: str) -> InstanceResources:
+    logger.info(f"fetching instance type details: {instance_type}")
```
Comment (ilkinmammadzada) [inline clusterman/aws/markets.py:406]: it will work for all requests not only for getting from aws. is it fine?

File: clusterman/aws/markets.py [hunk 2]
```diff
@@ -406,6 +412,7 @@ def get_instance_type(instance_type: str) -> InstanceResources:
         if instance_type not in EC2_INSTANCE_TYPES:
             raise ValueError(f"Invalid instance type: {instance_type}")
         else:
+            logger.info(f"located instance-type details: {EC2_INSTANCE_TYPES[instance_type]}")
```
Comment (ilkinmammadzada) [inline clusterman/aws/markets.py:415]: same as above

**Review by ilkinmammadzada — COMMENTED** [review]
LGTM! only one question

File: clusterman/aws/markets.py (hunk 1 above)
Comment (ajayOO8) [reply clusterman/aws/markets.py:406]: yeah, I wanted to check for both cases. Right now there are limited logs in this code flow, so couldn't verify the working. @ilkinmammadzada

**Review by ilkinmammadzada — APPROVED** [review]

**Pull request merged**